Python/Product/Profiling/PythonProfilingPackage.cs

```diff
@@ -286,13 +286,16 @@ protected override int CreateToolWindow(ref Guid toolWindowType, int id)
 
         private static void ProfileStandaloneTarget(SessionNode session, StandaloneTarget runTarget, bool openReport) {
             LaunchConfiguration config;
-            if (runTarget.PythonInterpreter != null) {
+            if (runTarget.PythonInterpreter != null && runTarget.PythonInterpreter.Id != "Global|Other...") {
                 var registry = session._serviceProvider.GetComponentModel().GetService<IInterpreterRegistryService>();
                 var interpreter = registry.FindConfiguration(runTarget.PythonInterpreter.Id);
                 if (interpreter == null) {
                     return;
                 }
                 config = new LaunchConfiguration(interpreter);
+            } else if (runTarget.PythonInterpreter.Id == "Global|Other...") {
```

Comment:
Same as above. Hardcoded strings are usually bad practice. This should be reading from a static string somewhere, and `Other...` should come from the localized strings.